main.cpp

```diff
@@ -126,7 +136,138 @@ i cubed: 531441
 Use a service like https://www.diffchecker.com/diff to compare your output. 
 */
 
-#include <iostream>
+
+template<class NumType>
+struct Numeric
+{
+    //using Type = Temporary<NumType>;
+    using Type = NumType;
+
+    Numeric(Type nuNum) : value( std::make_unique<Type>(nuNum) ) { } 
```

Comment:
When you change your `Type` alias, you need to think carefully about what type of value `Temporary` wants passed to its constructor.   **That type** is what type you need to use for this `Numeric` constructor parameter.
hint: it is NOT `Type`.
- [ ] revise